app/components/markdown/index.ts

```diff
@@ -14,11 +14,13 @@ import type {WithDatabaseArgs} from '@typings/database/database';
 const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
     const enableLatex = observeConfigBooleanValue(database, 'EnableLatex');
     const enableInlineLatex = observeConfigBooleanValue(database, 'EnableInlineLatex');
+    const maxNodes = observeConfigIntValue(database, 'MaxMarkdownNodes');
```

Comment:
Note that this value defaults to 0 both here and on the server. If it's 0, the renderer will use its own hardcoded default value. I realize now that this means that we'd need to either change this code or the renderer library if we want to change it, but regardless of that, it means we can tweak this without having to worry about updating the value stored in the config.json. Hopefully we won't need to change it, but if we happen to get reports of people actually hitting this limit during normal use, then we can address it without them needing to deploy a new server version.

Reply:
What if someone wants to remove the limit? Should they set to some special value (-1) or an absurdly high value?

Reply:
They'd just set it to be arbitrarily high. Outside of degenerate cases, I had a hard time getting above the default limit of 2000 though